common/common.mk

```diff
@@ -42,7 +42,7 @@ CFLAGS ?= -I$(LIBBPF_DIR)/root/usr/include/ -g
 CFLAGS += -I../headers/
 LDFLAGS ?= -L$(LIBBPF_DIR)
 
-LIBS = -lbpf -lelf
+LIBS = -lbpf -lelf -lpcap
```

Comment:
yes, it will be linked only to binaries that need it,

but we could have something like LIBS_EXTRA=-lpcap defined
in the Makefile and the common stuff would check for it